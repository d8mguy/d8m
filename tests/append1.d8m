// test for list.append method
val lst1 = [0, 4, 3, 4, 27]
val foo = \imp(lst : list(integer)) {
    var ret = copy(lst)
    ret.append(lst)
    ret
}
foo(lst1)
