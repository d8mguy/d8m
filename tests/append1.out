[0, 4, 3, 4, 27, 0, 4, 3, 4, 27]
